### ISSUE TITLE
Make valid only the current bar

### ISSUE THREAD

**Issue by saitodisse** [cirocosta/react-piano-tiles#8]
When clicking above, on future bars, it works, but it's impossible to miss.

**Comment by cirocosta** [comment]
:+1: didn't see that!

As far as i remember this issue might be solved by checking the constraints [in the Piano component](https://github.com/cirocosta/react-piano-tiles/blob/238c67b4fd08a9ee776b89e851c0716743458a55/src/components/Piano.jsx#L48-L58), which is the first check to valid clicks. Would you like to go for it @saitodisse ?

**Comment by saitodisse** [comment]
Yes I can do. But I think I'll do in the future because I am still learning Reac and not Flux. If you want to make this correction you can do, man. No hard feelings. ;)
